test/Sql/SelectTest.php

```diff
@@ -1397,4 +1399,21 @@ public function providerData()
             [$select52, $sqlPrep52, [],    $sqlStr52, $internalTests52],
         ];
     }
+
+    public function testPropagateAdapterWithGetSqlStringThatAlreadyBroughtBySqlObject()
+    {
+        if (extension_loaded('mysqli')) {
```

Comment:
I agree that it is much more work, however, i believe that the test is fubar without it and creates the unnecessary reliance on an extension inside of the adapter, if anything use the sqlite adapter with :memory: